python-sdk/langwatch/examples/migration_support/guardrails.py

```
@@ -0,0 +1,47 @@
+async def main(message: cl.Message):
+    langwatch.get_current_trace().autotrack_openai_calls(client)
+
+    msg = cl.Message(
+        content="",
+    )
+
+    jailbreak_guardrail = langwatch.get_current_span().evaluate(
+        "jailbreak-detection", as_guardrail=True, input=message.content
+    )
+    if not jailbreak_guardrail.passed:
+        await msg.stream_token(f"I'm sorry, I can't help you with that.")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)